docs/guide/software-preparation.md

```diff
@@ -74,7 +74,49 @@ For further guidance on the editor, feel free to contact us on [Twitter](https:/
 
 ### Image Editors
 
--   [Krita](https://krita.org/en/) _or_ [GIMP](https://www.gimp.org/) _or_ [paint.net](https://www.getpaint.net)
+When choosing an image editor, it is important to keep in mind that the traditional Minecraft style is composed of simple 16X16 pixelart. There are plenty of powerful and free art programs avialable for you to use. However, many of these programs have more tools than you will need for Minecraft graphical design and these tools require time to learn. 
+
+:::tip
+Choose a program that feels comfortable and easy for you to use. Many Addon creators use different art programs for different tasks. (Example: I use paint.net for most of the art, and piskel for Minecraft block animations). Choose what works best for you! 
+:::
+
+- **Krita**
+Krita is a powerful open-source art programed with the goal of giving free powerful digital art tools to artists. Krita has more than enough features to cover your Minecraft needs and works on a MAC or PC.
```

Comment:
```
on a MAC
```

Should we do this? The bedrock version doesn't work on a Mac, so will anyone code such addons on a MAC?

Reply:
I would also remove the "Example: I use", and replace with "One might use"